### PR TITLE
added content on restoring data

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira: https://issues.redhat.com/browse/ACM-7900

### CODE REVIEW

**Comment by jc-berger** [comment]
@birsanv , thanks for providing awesome content for me to work with! 

I made changes based off Red Hat style guides and language standards. Please let me know if I unintentionally altered meaning with my word changes and edits. Thanks!

File: business_continuity/backup_restore/backup_restore.adoc [hunk 1]
```diff
@@ -57,6 +57,13 @@ spec:
...
 
+[#restoring-data-primary-hub]
+== Restoring data back to the initial primary hub 
+
+When you need to use a passive hub cluster, create a new passive hub cluster. If you use an existing passive hub cluster that already has data, then the data on this hub cannot reflect the data available from the backup resources or the data after a completed restore. During the hub restore operation, you can configure the hub backup restore to update the existing resources if these resources are not part of the backup data being restored. The hub backup restore only updates the resources if they are resource types known to the backup. If the resources have a CRD that is unknown to the hub backup resource, then the hub backup restore ignores that CRD resource. The hub backup restore only updates resources created by an earlier backup; it cannot update user resources.
```
Comment (jc-berger) [inline business_continuity/backup_restore/backup_restore.adoc]: something like:

when you need to restore passive data on a cluster, create a new cluster

Comment (jc-berger) [reply business_continuity/backup_restore/backup_restore.adoc]: no need to always say passive. 

"if you use an exisitng hub cluster..."

see nots to help rewrite this section:

Velero updates existing resources if they have changed with the currently restored backup. It does not clean up delta resources, which are resources created by a previous restore and not part of the currently restored backup. This limits the scenarios that can be used when restoring hub data on a new hub. Unless the restore is applied only once, the new hub could not be relibly used as a passive configuration: the data on this hub is not reflective of the data available with the restored resourc

the new hub could not be relibly used as a passive configuration: the data on this hub is not reflective of the data available with the restored resourc

basically, the data is not reflective of what you'd expect on the hub. the hub needs to match the data exactly from the intial hub where the backup originally happened. 

scenario: if you have an exiting hub and your have 3 applications, and you want to use a backup restore hub, then you get the data from the restore and these three applications. So you have lots of data that's not matching the intital hub.

Comment (jc-berger) [inline business_continuity/backup_restore/backup_restore.adoc]: say "cleaned up" INSTEAD of "updated" Clean up is an actual option for the user!

Comment (jc-berger) [inline business_continuity/backup_restore/backup_restore.adoc]: last three sentences of the first paragraph could probably be condensed to one sentence: "The restore cleans up resources created by a previous backup but will not clean up user resources."

File: business_continuity/backup_restore/backup_restore.adoc [hunk 2]
```diff
@@ -57,6 +57,13 @@ spec:
...
+
+When you need to use a passive hub cluster, create a new passive hub cluster. If you use an existing passive hub cluster that already has data, then the data on this hub cannot reflect the data available from the backup resources or the data after a completed restore. During the hub restore operation, you can configure the hub backup restore to update the existing resources if these resources are not part of the backup data being restored. The hub backup restore only updates the resources if they are resource types known to the backup. If the resources have a CRD that is unknown to the hub backup resource, then the hub backup restore ignores that CRD resource. The hub backup restore only updates resources created by an earlier backup; it cannot update user resources.  
+
+Use an existing passive hub cluster during a disaster recovery test. In a recovery test, you are only testing the hub backup scenario. Therefore, you want to go back to the primary passive hub cluster after it restored the backup data. In this situation, the initial primary hub has not created new resources. Instead, the backup data has temporarily changed side  from the primary hub to the passive hub cluster.
```
Comment (jc-berger) [inline business_continuity/backup_restore/backup_restore.adoc]: first sentence should be somethign like: 

a disaster recoery test is an example situation where a user would use an exisitng hub cluster for a restore operation.

Comment (jc-berger) [reply business_continuity/backup_restore/backup_restore.adoc]: remove the "passive" :)

File: business_continuity/backup_restore/backup_restore.adoc (hunk 1 above)
Comment (jc-berger) [reply business_continuity/backup_restore/backup_restore.adoc]: for more help, see the readme for cleaning up hub restore: https://github.com/stolostron/cluster-backup-operator?tab=readme-ov-file#cleaning-up-the-hub-before-restore

**Comment by jc-berger** [comment]
THanks, @birsanv for your awesome feedback. I've kept my comments in case I need to refer back to them. Otherwise, the new commit is up for review. Thanks!

File: business_continuity/backup_restore/backup_restore.adoc [hunk 3]
```diff
@@ -57,6 +57,13 @@ spec:
...
+
+A disaster recovery test is an example situation where you would use an existing hub cluster for a restore operation.  In a recovery test, you are only testing the hub backup scenario. Therefore, you want to go back to the primary passive hub cluster after it restored the backup data. In this situation, the initial primary hub has not created new resources. Instead, the backup data has temporarily changed side  from the primary hub to the passive hub cluster.
```
Comment (birsanv) [inline business_continuity/backup_restore/backup_restore.adoc]: change A disaster recovery test is an example situation where you would use an existing hub cluster
with A disaster recovery test is an example situation where you **could** use an existing hub cluster

Comment (birsanv) [reply business_continuity/backup_restore/backup_restore.adoc]: remove : Therefore, you want to go back to the primary passive hub cluster after it restored the backup data.

File: business_continuity/backup_restore/backup_restore.adoc [hunk 4]
```diff
@@ -57,6 +57,13 @@ spec:
...
 
+[#restoring-data-primary-hub]
+== Restoring data back to the initial primary hub 
+
+When you need to restore passive data on a cluster, create a new cluster. Velero updates your existing resources if they have changed with the currently restored backup. Velero does not clean up `delta` resources, which are resources created by an earlier restore and not part of the currently restored backup. Because Velero does not clean up `delta` resources, you have less scenarios that you can use when you restore your hub data on a new hub. Unless you apply the restore only one time, you cannot reliably use the new hub as a passive configuration. As a result, the data on this hub is not reflective of the data available with the restored resource. During the hub restore operation, you can configure the hub backup restore to clean up the existing resources if these resources are not part of the backup data being restored. The restore cleans up resources created by an earlier backup but does not clean up user resources.
```
Comment (birsanv) [inline business_continuity/backup_restore/backup_restore.adoc]: I would change this paragraph with this 

When you need to restore backup data on a cluster, create a new cluster. During the hub restore operation, you can configure the hub backup restore to clean up the existing resources if these resources are not part of the backup data being restored. The restore cleans up resources created by an earlier backup but does not clean up user resources. As a result, resources created by the user on this hub will not be cleaned up so the on this hub is not reflective of the data available with the restored resource.

**Review by birsanv — CHANGES_REQUESTED** [review]

File: business_continuity/backup_restore/backup_restore.adoc [hunk 5]
```diff
@@ -57,6 +57,13 @@ spec:
...
 
+[#restoring-data-primary-hub]
+== Restoring data back to the initial primary hub 
+
+When you need to restore backup data on a cluster, create a new cluster. During the hub restore operation, you can configure the hub backup restore to clean up the existing resources if these resources are not part of the backup data being restored. The restore cleans up resources created by an earlier backup but does not clean up user resources. As a result, resources created by the user on this hub will not be cleaned up, so the data on this hub is not reflective of the data available with the restored resource.
+
+A disaster recovery test is an example situation where you could use an existing hub cluster. In a recovery test, you are only testing the hub backup scenario. In this situation, the initial primary hub has not created new resources. Instead, the backup data has temporarily changed side  from the primary hub to the passive hub cluster.
```
Comment (dockerymick) [inline business_continuity/backup_restore/backup_restore.adoc]:
```suggestion
When you need to restore backup data on a cluster, create a new cluster. During the hub cluster restore operation, you can configure the hub cluster backup restore to clean up the existing resources, if these resources are not part of the backup data being restored. The restore cleans up resources created by an earlier backup but does not clean up user resources. As a result, resources created by the user on this hub cluster are not cleaned up, so the data on this hub cluster is not reflective of the data available with the restored resource.

A disaster recovery test is an example situation where you can use an existing hub cluster. In a recovery test, you are only testing the hub backup scenario. In this situation, the initial primary hub cluster does not create new resources. Instead, the backup data has temporarily changed sides from the primary hub cluster to the passive hub cluster.  
```

**Review by dockerymick — APPROVED** [review]
made a suggestion

**Comment by dockerymick** [comment]
@jc-berger just noticed that this is going into 2.9_prod branch. Please close this PR and create one for 2.9_stage. The base branch updated automatically after i merged my PR
Screenshot 2024-01-30 at 10 08 15 AM

I'm not sure why that happened. We can bring it up during the scrum

**Review by dockerymick — CHANGES_REQUESTED** [review]
Removing approval to bring notice to the base branch change. This was complete due to an automation. Jake is on PTO today and will make updates throughout the week. Thanks!

**Comment by swopebe** [comment]
No not merge this to PROD, but to STAGE

**Review by birsanv — APPROVED** [review]
lgtm

**Comment by jc-berger** [comment]
Due to complications with this PR to prod, I will close it. 

See the content in the new PR for 2.10_stage: https://github.com/stolostron/rhacm-docs/pull/6083
